### ISSUE TITLE
Json output on windows - path issues

### ISSUE THREAD

**Issue by OlivierMasit** [lunasec-io/lunasec#365]
On Windows the path contains double \ instead of a single one. c:\\folder\\lib instead or c:\folder\lib.
```Json
 {
                        "path": "c:\\Users\\oli\\Downloads\\test.war::WEB-INF/lib/log4j-core-2.13.3.jar",
                        "file_name": "org/apache/logging/log4j/core/net/JndiManager.class",
                        "hash": "c3e95da6542945c1a096b308bf65bbd7fcb96e3d201e5a2257d85d4dedc6a078",
                        "version": "2.13.0, 2.13.1, 2.13.2, 2.13.3",
                        "cve": "CVE-2021-44228",
                        "severity": "10.0"
},

**Comment by 1tft** [comment]
@OlivierMasit No bug/isse.
Backspaces has to be escaped with backspace to produce valid json: https://www.freeformatter.com/json-escape.html

**Comment by OlivierMasit** [comment]
> @OlivierMasit No bug/issue. Backspaces has to be escaped with backspace to produce valid json: https://www.freeformatter.com/json-escape.html

Sure! I just wanted to report it so it can be done in the code 😉

**Comment by freeqaz** [comment]
I'm not sure if I understand this issue correctly. Per what @1ftf said, this seems like the correct behavior for the JSON output to contain escaped backslashes.

What exactly would you expect the output to be? How should we resolve this?

**Comment by OlivierMasit** [comment]
> I'm not sure if I understand this issue correctly. Per what @1ftf said, this seems like the correct behavior for the JSON output to contain escaped backslashes.
> 
> What exactly would you expect the output to be? How should we resolve this?

the console output is doing it event when not in json.
```shell
10:56AM Scan Result: Identified vulnerable path
        cve: CVE-2021-44228
        fileName: org/apache/logging/log4j/core/net/JndiManager.class
        hash: c3e95da6542945c1a096b308bf65bbd7fcb96e3d201e5a2257d85d4dedc6a078
        path: "C:\\Users\\Oli\\Downloads\\ncbe-tomcat-21.11.03-SNAPSHOT.war::WEB-INF/lib/log4j-core-2.13.3.jar"
        severity: 10.0
        versionInfo: "2.13.0, 2.13.1, 2.13.2, 2.13.3"
```
the path should be C:\Users\Oli\Downloads\ncbe-tomcat-21.11.03-SNAPSHOT.war

**Comment by freeqaz** [comment]
Hey @OlivierMasit,

Thanks for clarifying about this! I don't have time to deal with this specific issue today (or likely this week with the holidays). I added that tag, "good first issue", in the hopes that somebody else might be able to take a look at changing this.

Cheers,
- Free

**Comment by Emusp** [comment]
Is there any way the windows version of the scan tool could output the result to a json file like the cli tools does?

**Comment by OlivierMasit** [comment]
> Is there any way the windows version of the scan tool could output the result to a json file like the cli tools does?

Yes, check my help here. I'll try to to do it better and create PR.
https://github.com/lunasec-io/lunasec/discussions/364

**Comment by breadchris** [comment]
Fixed in 1.5.1
